biorxiv/pca_association_experiment/biorxiv_pca_category_bootstrap.py

```diff
@@ -339,3 +344,225 @@
 g.save("output/pca_plots/figures/category_box_plot_pc2.svg", dpi=500)
 print(g)
 
+
+# # Tables with Figures
+
+# In[17]:
+
+
+def dump_figures_and_table(
+    table_df, figure_selector, 
+    output_path="output/table", 
+    column_value="PC_1"
+):
+    
+    # Output figures to folder
+    for idx, row in table_df.iterrows():
+        subprocess.Popen([
+            "unzip", "-j",
+            Path("..")/Path(row['hash']), 
+            Path("content")/Path(figure_selector[row['document']]),
+            "-d", Path("output/table_figures")
+        ])
+        
+    # Output table
+    (
+        table_df
+        [["document", "doi", column_value]]
+        .assign(
+            doi=lambda x: x.doi.apply(lambda link: f"[@doi:{link}]"),
+            figure=lambda x: x.document.apply(lambda doc: f"![](table_figures/{document_figure_selector[doc]})")
+        )
```

Comment:
It would be helpful to have some comments on this part, it took awhile to follow what the columns being created were

Reply:
I think I'm trying to understand what you mean by "Bottom PC2" here. Why are you interested in those documents that have a low PC2 score? I assume this is because these are documents that have low similarity with PC2..?

Is there a reason you chose to look at systems biology as a first pass?

Reply:
>I think I'm trying to understand what you mean by "Bottom PC2" here. Why are you interested in those documents that have a low PC2 score? I assume this is because these are documents that have low similarity with PC2..?

This idea here is to provide a visual confirmation for the concepts captured by PC2. The bottom PC2 images here are figures that would help indicate that PC2's negative direction is neuroscience. 

>Is there a reason you chose to look at systems biology as a first pass?

Systems biology was picked because the field is pretty broad in terms of research type. Experiments can range from simple wet lab experiments to full on computational results. This category was picked during my one-on-one meetings with Casey.